etc/picongpu/summit-ornl/gpu_picongpu.profile.example

```diff
@@ -18,23 +18,22 @@ export proj=<yourProject>
 #export EDITOR="nano"
 
 # basic environment ###########################################################
-module load gcc/8.1.1
+module load gcc/6.4.0  e4s/21.02  spectrum-mpi/10.3.1.2-20200121
```

Comment:
Is there no newer `gcc` compatible with all dependencies we need? I do not know of any particular issues with 6.4.0, just it is 4 years old

Reply:
Yes, this is exactly the point. The other available option would be gcc/9.3.0. But there is no boost with this gcc available that fits our requirements. I could ask for it though. Will do. But for now we can merge this, can't we?

Reply:
Okay

Reply:
On the other hand. I just saw that our readthedocs recommends
```
CUDA 10.1/10.2: Use gcc 5.5 - 8
```
Not sure where this is coming from, but with respect to this I would say that everything is fine the way it is as the only alternative to gcc 6 is gcc 9.

Reply:
Could you ask for a gcc 8?